public/scss/components/OrderSummaryBox.module.scss

```
@@ -747,6 +747,7 @@
 
     &_voucherDetailCode
     {
+        display: none;
         @include typographyBuilder(secondary, 400, 12, 16);

```

```suggestion
    {
        @include typographyBuilder(secondary, 400, 12, 16);
        display: none;

```

include taro paling atas kaks